src/Model/ContextInterface.php

```diff
@@ -44,16 +44,16 @@ public function setEnabled($enabled);
     public function getEnabled();
 
     /**
-     * Set slug.
+     * Set id.
      *
-     * @param int $id
+     * @param string $id
      */
     public function setId($id);
 
     /**
-     * Get slug.
+     * Get id.
```

Comment:
And that too. We learn nothing here.

Reply:
@greg0ire agree, but it should be done on the entire project. What about another PR in the future, leaving it as-is for now?

Reply:
As you wish